### PR TITLE
Change max avatar size to match other platforms

### PR DESCRIPTION
**Pull request by Aerilym**
Before:
[Screencast from 2024-05-02 11-21-59.webm](https://github.com/oxen-io/session-desktop/assets/5667907/d4d0d333-3d20-44a0-ad4b-7512096da13d)

After:

Screenshot from 2024-05-02 14-11-55
Also change the max avatar file size to 5MB to be in line with IOS and Android.

### CODE REVIEW

File: ts/util/attachmentsUtil.ts [hunk 1]
```diff
@@ -53,7 +54,7 @@ export async function autoScaleForAvatar<T extends { contentType: string; blob:
 ) {
   const maxMeasurements = {
     maxSide: AVATAR_MAX_SIDE,
-    maxSize: 1000 * 1024,
+    maxSize: 5 * 1024 * 1024,
```
Comment (yougotwill) [inline ts/util/attachmentsUtil.ts:57]: Might be nice to have a constant for this. See usage of `DURATIONS` as an example.

```tsx
export const FILESIZES = {
  /** 1KB */
  KB: 1024,
  /** 1MB */
  MB: 1024 * 1024,
  /** 1GB */
  GB: 1024 * 1024 * 1024,
};

You can choose a better name if you can think of one
```

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/util/attachmentsUtil.ts (hunk 1 above)
Comment (yougotwill) [reply ts/util/attachmentsUtil.ts:57]: and replace any other file sizes in the project please

Comment (Bilb) [reply ts/util/attachmentsUtil.ts:57]: I'd argue that a constant used only in one file should stay in that file (but still a const at the top)

**Review by yougotwill — APPROVED** [review]
Approved! ✅

**Comment by Bilb** [comment]
regressions tests passed so merging this in.

**Pull request merged**